app/assets/stylesheets/application.css.scss

```diff
@@ -13,3 +13,40 @@
  *= require_tree .
  *= require_self
  */
+
+ #container {
+   margin: 48px auto;
+   width: 960px;
+
+   > {
+     section {
+       clear: both;
+     }
+
+     header {
+       margin-bottom: 46px;
+
+       > h1 {
+         float: left;
+         margin: 0;
+       }
+     }
+   }
+ }
+
+ .clearfix:after {
+   clear: both;
+   content: ".";
+   display: block;
+   height: 0;
+   line-height: 0;
+   visibility: hidden;
+ }
+
+ html[xmlns].clearfix {
+   display: block;
+ }
+
+ * html .clearfix {
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Selector should have depth of applicability no greater than 2, but was 3